sdk/keyvault/keyvault/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required (VERSION 3.0)
```

Comment:
This minimum version doesn't match what's in the template https://github.com/Azure/azure-sdk-for-c/pull/2/files (3.15)

Reply:
Yes, I started it before. I will update to the latest template as soon as it merged.

Reply:
this should be at least 3.12, no reason to revert to a lower version here after setting 3.12 in the parent project